UPGRADE-4.2.md

```diff
@@ -75,6 +75,8 @@ Process
    // alternatively, when a shell wrapper is required
    $process = Process::fromShellCommandline('ls -l');
    ```
+ * Add trigger_error messages with E_USER_DEPRECATED to functions in components Bridge/Monolog, BrowserKit, DomCrawler, Finder and Serializer when class is inherited and
```

Comment:
there should be one line per affected component, telling roughly the same as what deprecation messages tell (same in UPGRADE-5.0)